content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml

```diff
@@ -114,6 +114,48 @@
                             </column>
                         </items>
                     </socialmedia>
+                    <singlepageapps
+                        cq:showOnCreate="{Boolean}true"
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Single Page Apps"
+                        sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                        sling:orderBefore="cloudservices">
+                        <items jcr:primaryType="nt:unstructured">
+                            <column
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/container">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <isroot
+                                        cq:showOnCreate="{Boolean}true"
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                        name="./isroot"
+                                        renderReadOnly="{Boolean}true"
+                                        text="Is Root"
```

Comment:
We should add a `fieldDescription`

Ex:

`The current page is the root of a hierarchy`